### PR TITLE
ADD Missing operations to doc

### LINKED ISSUE #145: There are URLS allowed, not documented
There are some URLs implemented in the OrionPlugin, that are not documented in the doc, 

```
  ['GET', /^\/v1\/contextentities/, 'read'],
  ['POST', /^\/v1\/contextentities/, 'create'],
  ['GET', /^\/v1\/contextsubscriptions/, 'read'],
  ['GET', /^\/v1\/contextsubscriptions\/.+/, 'read'],
  ['GET', /^\/v1\/contexttypes/, 'read'],
  ['GET', /^\/v1\/contexttypes\/.+/, 'read']
```

### PR DESCRIPTION
**Pull request by dmoranj**
Fix #145

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -566,6 +566,8 @@ An up-to-date list of the convenience operations can be found [here](https://doc
 #### NGS10 (context information availability)
 | Method | Path                                                                                     | Action |
 | ------ |:--------------------------------------------------------------------------------------- | ---:|
+| GET    | /v1/contextentities                                                                  | R |
```
Comment (fgalan) [inline README.md]: I understand that PEP is not case sensitive but maybe for coherence in this table you should use the usual style in CB operations. i.e.:

```
/v1/contextEntities
```

BTW, I'm almost sure that Orion is case sensitive in operations ( @kzangeli could confirm definetively)

File: README.md [hunk 2]
```diff
@@ -587,8 +589,12 @@ An up-to-date list of the convenience operations can be found [here](https://doc
 | GET    | /v1/contextEntityTypes/{typeName}/attributes/{attributeName}                       	| R |
 | GET    | /v1/contextEntityTypes/{typeName}/attributeDomains/{attributeDomainName}           	| R |
 | POST   | /v1/contextSubscriptions                                                           	| S |
+| GET    | /v1/contextsubscriptions                                                             | R |
```
Comment (fgalan) [inline README.md]: contextSubscriptions (with capitalized 'S')

File: README.md (hunk 1 above)
Comment (dmoranj) [inline README.md]: Fixed in  f7ee70e

File: README.md (hunk 2 above)
Comment (dmoranj) [inline README.md]: Fixed in  f7ee70e

**Comment by fgalan** [comment]
LGTM

**Pull request merged**